azure/template.json

```
@@ -201,6 +202,47 @@
             "dependsOn": [
                 "[variables('resourceGroupName')]"
             ]
+        },
+        {
+            "apiVersion": "2019-05-10",
```

```suggestion
            "apiVersion": "2021-04-01",
```
To be consistent with other resource deployments